tests/MetricExporter.spec.ts

```diff
@@ -305,7 +305,7 @@ describe("MetricExporter.export", () => {
 				metricKind: MetricKind.UP_DOWN_COUNTER,
 				valueType: ValueType.DOUBLE
 			},
-			labels: labels,
+			attributes: attributes,
```

Comment:
Please update the README and sample.js which still use `labels` in some places.

Reply:
updated sample but i dont see use of labels in readme

Reply:
ah nvm found and fixed